<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic;
}

QMainWindow
{
background-image: url(:/bg/BG.webp);
/* Full height */
height: 100%;
background-position: center; /* Center the image */
background-repeat: no-repeat; /* Do not repeat the image */
 background-size: cover;
}

QProgressBar
{
border:none;
border-radius: 10px;
border: 1px solid grey;
text-align: center
}

QProgressBar::chunk {
    background-color: #4CC417;
    margin-left: 0px;
	border-radius: 8px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">QFrame
{
	background:rgba(0, 0, 0, 0.8);
	border-radius:10px;
	opacity: 0.5;
}

QLabel
{
color:white;
font-size:16px;
background:transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Analisis Sentimen Game Genshin Impact</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame
{
	background:rgba(0, 0, 0, 0.6);
	border-radius:10px;
	opacity: 0.5;
}

QPushButton
{
border-radius:15px;
font-size:16px;
color: white;
background:transparent;
}


QToolButton
{
background:transparent;
border-radius:40px;
background:transparent;
}

QLabel
{
color:white;
font-size:16px;
background:transparent;
}

QLineEdit
{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}

QTableView
{
background:transparent;
border:none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QPushButton" name="btnPilihFile">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:5px;
font-size:14px;
color: white;
background:rgba(116, 201, 29, 0.8);
}

QPushButton:hover {
  background:rgba(65, 142, 28, 0.8);;
  color: white;
}

QPushButton:disabled,
QPushButton[disabled]{
 color: white;
 background:rgba(225, 234, 221, 0.8);;
}

</string>
         </property>
         <property name="text">
          <string>Pilih File ...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtPilihFile">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>century gothic</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Upload Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">QFrame
{
	background:rgba(255, 255, 255, 0.8);
	border-radius:10px;
	opacity: 0.5;
}

QPushButton
{
border-radius:15px;
font-size:18px;
color: white;
background:transparent;
}


QToolButton
{
background:transparent;
border-radius:40px;
background:transparent;
}

QLabel
{
color:black;
font-size:12px;
background:transparent;
}

QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QTableView
{
background:transparent;
border:none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableViewData">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">QFrame
{
	background:rgba(0, 0, 0, 0.6);
	border-radius:10px;
	opacity: 0.5;
}

QPushButton
{
border-radius:5px;
font-size:14px;
color: white;
background:rgba(209, 67, 192, 0.8);;
}

QToolButton
{
background:transparent;
border-radius:40px;
background:transparent;
border-radius:5px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <widget class="QPushButton" name="btnProses">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
   
  background:rgba(134, 13, 120, 0.8);;
  color: white;
}

QPushButton:disabled,
QPushButton[disabled]{
 color: white;
 background:rgba(213, 204, 212, 0.8);;
}</string>
         </property>
         <property name="text">
          <string>PROSES</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnHasilSVM">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:5px;
font-size:14px;
color: white;
background:rgba(116, 201, 29, 0.8);;
}

QPushButton:hover {
  background:rgba(65, 142, 28, 0.8);;
  color: white;
}

QPushButton:disabled,
QPushButton[disabled]{
 color: white;
 background:rgba(225, 234, 221, 0.8);;
}
</string>
         </property>
         <property name="text">
          <string>Hasil SVM</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnHasilPreprocessing">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:5px;
font-size:14px;
color: white;
background:rgba(133, 243, 20, 0.8);
}

QPushButton:hover {
  background:rgba(65, 152, 26, 0.8);
  color: white;
}

QPushButton:disabled,
QPushButton[disabled]{
 color: white;
 background:rgba(225, 234, 221, 0.8);
}
</string>
         </property>
         <property name="text">
          <string>Hasil Preprocessing</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnHasilTFIDF">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:5px;
font-size:14px;
color: white;
background:rgba(159, 248, 69, 0.8);
}

QPushButton:hover {
  background:rgba(134, 203, 62, 0.8);
  color: white;
}

QPushButton:disabled,
QPushButton[disabled]{
 color: white;
 background:rgba(225, 234, 221, 0.8);
}
</string>
         </property>
         <property name="text">
          <string>Hasil TFIDF</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="pB">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
